proto/regen/ecocredit/marketplace/v1beta1/state.proto

```diff
@@ -4,55 +4,58 @@ package regen.ecocredit.marketplace.v1beta1;
 
 import "google/protobuf/timestamp.proto";
 import "cosmos/orm/v1alpha1/orm.proto";
-import "cosmos/base/v1beta1/coin.proto";
 import "regen/ecocredit/marketplace/v1beta1/types.proto";
 
 // SellOrder represents the information for a sell order.
 message SellOrder {
     option (cosmos.orm.v1alpha1.table) = {
         id: 1,
-        primary_key: {fields: "order_id"}
-        index: {id: 1 fields: "batch_denom"}
-        index: {id: 2 fields: "owner"}
+        primary_key: {fields: "id", auto_increment: true}
+        index: {id: 1 fields: "batch_id"}
+        index: {id: 2 fields: "seller"}
         index: {id: 3 fields: "expiration"}
     };
 
-    // order_id is the unique ID of sell order.
-    uint64 order_id = 1;
+    // id is the unique ID of sell order.
+    uint64 id = 1;
 
-    // owner is the address of the owner of the credits being sold.
-    string owner = 2;
+    // seller is the address of the owner of the credits being sold.
+    bytes seller = 2;
 
-    // batch_denom is the credit batch being sold.
-    string batch_denom = 3;
+    // batch_id is ID of the credit batch being sold.
+    uint64 batch_id = 3;
 
     // quantity is the quantity of credits being sold.
     string quantity = 4;
 
-    // ask_price is the price the seller is asking for each unit of the
+    // market_id is the market in which this sell order exists and specifies
+    // the back denom ask_price corresponds to.
+    uint64 market_id = 5;
+
+    // ask_price is the integer price the seller is asking for each unit of the
     // batch_denom. Each credit unit of the batch will be sold for at least the
```

Comment:
The description says it is integer, but defined as string

Reply:
it's an integer, but stored as a string like in bank